buildSrc/build.gradle.kts

```diff
@@ -2,12 +2,6 @@ plugins {
     `kotlin-dsl`
 }
 
-java {
-    toolchain {
-        languageVersion.set(JavaLanguageVersion.of(17))
```

Comment:
This is here for our linting tools etc, the build still runs on Java 8, see `buildSrc/src/main/kotlin/java-conventions.gradle.kts`